Recrutify/Recrutify.DataAccess/Repositories/Abstract/IScheduleRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Recrutify.DataAccess.Models;
+
+namespace Recrutify.DataAccess.Repositories.Abstract
+{
+    public interface IScheduleRepository
+    {
+        Task<List<Schedule>> GetUsersSchedulesByPrimarySkillAsync(IEnumerable<Guid> interviewersIds,  DateTime date, Guid primarySkillId);
```

Comment:
usersIds

Reply:
+

Reply:
think it's enougth to name GetByUserPrimarySkillAsync

Reply:
+